pyresample/test/test_bucket.py

```diff
@@ -107,44 +106,44 @@ def test_get_bucket_indices(self):
         np.testing.assert_equal(resampler.x_idxs, np.array([-1, 0, 0, 1, 1, 1, -1, -1, -1]))
         np.testing.assert_equal(resampler.y_idxs, np.array([-1, 1, 1, 1, 0, 0, -1, -1, -1]))
 
+    def _get_sum_result(self, data, **kwargs):
+        """Compute the bucket average with kwargs and check that no dask computation is performed."""
+        with dask.config.set(scheduler=CustomScheduler(max_computes=0)):
+            result = self.resampler.get_sum(data, **kwargs)
+        return result.compute()
+
     def test_get_sum(self):
         """Test drop-in-a-bucket sum."""
-        data = da.from_array(np.array([[2., 2.], [2., 2.]]),
+        data = da.from_array(np.array([[2., 3.], [7., 16.]]),
```

Comment:
I would split `test_get_sum` into multiple test functions, what do you think?

Reply:
Good idea! Just added a commit, I also split the tests for `get_average`

Reply:
Looks great!